test/Spec/MockContract.hs

```diff
@@ -326,21 +326,21 @@ mockCallCommand ShellArgs {cmdName, cmdArgs, cmdOutParser} = do
     ("cardano-cli", "transaction" : "build-raw" : args) -> do
       case drop 1 $ dropWhile (/= "--out-file") args of
         filepath : _ ->
-          modify @(MockContractState w) (files . at (Text.unpack filepath) ?~ OtherFile "TxBody")
+          modify @(MockContractState w) (files . at (Text.unpack filepath) ?~ _)
```

Comment:
Tests are currently failing as we didn't previously care for the contents of these files but now we want to read them back to return the Tx

Reply:
@gege251 do you have any idea what the best solution for this could be? Atm `mockReadFileTextEnvelope` fails because these are stored as `OtherFile`, to store them as TextEnvelopeFiles I need to serialise an actual tx into it, what do you think is the best approach here?

Reply:
A bit hacky idea, but what if we would create a new `TxFile` constructor for `MockFile`, and use that in the pattern match for it in `mockReadFileTextEnvelope`, and return some dummy CardanoTx with some random id (I think, we don't need it to be correct at this point).

Reply:
this might be easier said that done.

Reply:
https://github.com/mlabs-haskell/bot-plutus-interface/pull/84